Taskfile.yml

```diff
@@ -256,12 +259,14 @@ tasks:
     cmds:
       - "rm -rf '{{.OUTPUT_DIR}}'"
       - for:
-          - "client"
-          - "yscope-log-viewer"
+          - PATH: "client"
+            OUTPUT_DIR_FLAG: "--emptyOutDir --outDir"
+          - PATH: "yscope-log-viewer"
+            OUTPUT_DIR_FLAG: "--output-path"
```

Comment:
@kirkrodrigues we can use the same build options from the `client` for yscope-log-viewer` once https://github.com/y-scope/yscope-log-viewer/pull/175 is merged